enhancements/sig-architecture/26-policy-generator/README.md

```diff
@@ -0,0 +1,266 @@
+4. As a k8s admin, I want to convert existing Compliance Operator scans into ACM policies.
+5. As a security/compliance focal, I want to be able to specify standards that a given policy
+   applies to.
+
+### Architecture
+
+Kustomize is a template-free way to customize Kubernetes resource object manifests and is considered
+an industry standard. Kustomize has the concept of generators, which generate resource object
+manifests based on the input generator configuration. In order to leverage this concept, a Kustomize
+generator plugin will be written that accepts policy configuration and generates a policy with one
+or more ConfigurationPolicy objects.
+
+Through Kustomize, users will be able to override any defaults or add/remove fields to the generated
+policy by altering the generator plugin patch and/or supplying Kustomize patches. For example, if a
+user wants to generate a policy that creates a Kubernetes resource object (e.g. ConfigMap) on
+managed clusters, a security engineer may want to override the default policy standards annotation.
+Additionally, an SRE may want to add additional labels to the policy so that it is easily
+identified. These customizations can be in separate Kustomize patch files per role or combined in a
+single one. This flexibility allows the workflow to reflect an organization’s standards and
+structure.
+
+### User Experience
+
+#### The Kustomization YAML File
+
+Kustomize is configured with a `kustomization.yaml` file. To use the generator plugin, a user must
+declare it under the generators section as shown in a `kustomization.yaml` file below:
+
+```yaml
+generators:
+  - policyGenerator.yaml
+patchesStrategicMerge:
+  - input/patch.yaml
+commonLabels:
+  custom: myApp
+```
+
+The "policyGenerator.yaml" file is the generator configuration file that Kustomize will provide. All
+other customizations such as "patches" and "commonLabels" apply on the generated policy returned
+from the new generator plugin.
+
+#### Generator Defaults
+
+By default, the generator plugin will fill in some default policy fields so that it’s easier for the
+user when they are getting started.
+
+The following policy fields will be required:
+
+- name
+- namespace
+
+The following policy fields will be optional:
+
+- annotations
+- policy.open-cluster-management.io/categories - defaults to "CM Configuration Management"
+- policy.open-cluster-management.io/controls - defaults to "CM-2 Baseline Configuration"
+- policy.open-cluster-management.io/standards - defaults to "NIST SP 800-53" (to be confirmed)
+- disabled - defaults to "false"
+- remediationAction - defaults to "inform"
+- severity - defaults to "low"
+
+#### The Generator Configuration File
+
+As shown above in the example `kustomization.yaml` file, a configuration file for the generator
+plugin was provided in the "policyGenerator.yaml" file. The following example explains what each
+field is used for.
+
+```yaml
```

Comment:
Could you please replace this with the reference policy generator config file that you wrote?

Reply:
Yes--I was debating whether linking from upstream to our existing repo made sense, but I might as well use it since it's available!

Reply:
@dhaiducek I see what you mean. I was thinking we could just copy-paste the contents of the file here.

Reply:
Haha--oh, right. 🙂 I'll fix it.